css/cssom-1/cssom-ruleOrder.html

```diff
@@ -16,6 +17,7 @@
   <style id="s-2">
     h1 { background: indianred; }
   </style>
+  <!-- Unsure why the second of 2 rules is the test subject in #s-3 -->
```

Comment:
The svg namespace is already defined in stylesheet `#s-0`, was thinking that it didn't need to be declared again. But this may be my test-writing-newness showing?